cmd/shadowsocks-server/server.go

```diff
@@ -404,6 +413,7 @@ func main() {
 	flag.StringVar(&cmdConfig.Method, "m", "", "encryption method, default: aes-256-cfb")
 	flag.IntVar(&core, "core", 0, "maximum number of CPU cores to use, default is determinied by Go runtime")
 	flag.BoolVar((*bool)(&debug), "d", false, "print debug message")
+	flag.BoolVar((*bool)(&sanitizeIps), "noip-log", false, "suppress client ip addresses in all output")
```

Comment:
noip-log is not a good flag name

Reply:
@arthurkiller  Which do you prefer?

* `sanitize-ips`
* `no-ips`
* `privacy-log`

Reply:
I think `xxx-xxx` is not a good idea. maybe `-A` means anonymity or `-anonymity` is better

Reply:
after this, you'd better resolve the conflicts locally

Reply:
@arthurkiller Ok, I''ll change it to `-A` The idea is to not change the output too much to make log parsing scripts difficult to write.

What would you prefer instead of `x.x.x.x:zzzz`?